buildSrc/src/main/java/com/osacky/flank/gradle/FlankGradlePlugin.kt

```diff
@@ -69,17 +70,17 @@ class FlankGradlePlugin : Plugin<Project> {
       description = "Finds problems with the current configuration."
       group = TASK_GROUP
       workingDir("${project.fladleDir}/")
-      commandLine("java", "-jar", "flank.jar", "firebase", "test", "android", "doctor")
-      dependsOn(named("downloadFlank"), writeConfigProps)
+      commandLine("java", "-jar", project.fladleConfig.singleFile.absolutePath, "firebase", "test", "android", "doctor")
```

Comment:
Can't you instead set the class path to the configuration and then use args instead? Then you should not need to use the path to the jar directly